decidim-participatory_processes/app/cells/decidim/participatory_processes/process_g_cell.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcesses
+    # This cell renders the Grid (:g) process card
+    # for an given instance of a Process
+    class ProcessGCell < Decidim::CardMCell
```

Comment:
This seems to be a copy of ProcessMCell.

Reply:
As in the previous comment, the [correct version is in the card-g PR](https://github.com/decidim/decidim/blob/0c90b0a66dd9b61651f2fb38390f6c1d293717f4/decidim-participatory_processes/app/cells/decidim/participatory_processes/process_g_cell.rb)